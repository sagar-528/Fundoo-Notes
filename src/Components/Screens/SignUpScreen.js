import React from 'react'
import SignUp from '../SignUp'

const SignUpScreen = ({ navigation }) => {
    return (
        <SignUp navigation = {navigation}/>
    )
}

export default SignUpScreen
