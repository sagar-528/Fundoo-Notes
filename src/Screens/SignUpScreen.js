import React from 'react'
import SignUp from '../Components/Authentication/SignUp'

const SignUpScreen = ({ navigation }) => {
    return (
        <SignUp navigation = {navigation}/>
    )
}

export default SignUpScreen
