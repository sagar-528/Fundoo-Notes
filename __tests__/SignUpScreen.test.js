import React from 'react';
import {shallow} from 'enzyme';
import SignUpScreen from '../src/Screens/SignUpScreen';

describe('test SignUpScreen', () => {
    it('should match to snapshot', () => {
        const component = shallow(<SignUpScreen/>)
        expect(component).toMatchSnapshot();
    })
})