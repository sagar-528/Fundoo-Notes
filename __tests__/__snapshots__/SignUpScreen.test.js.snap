// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test SignInScreen should match to snapshot 1`] = `ShallowWrapper {}`;

exports[`test SignUpScreen should match to snapshot 1`] = `ShallowWrapper {}`;
